server/src/main/java/org/apache/gravitino/server/web/rest/PolicyOperations.java

```diff
@@ -96,14 +102,26 @@ public Response listPolicies(
                   Arrays.stream(policies)
                       .map(p -> toDTO(p, Optional.empty()))
                       .toArray(PolicyDTO[]::new);
-
+              policyDTOs =
```

Comment:
Could u add annotation for the list operations like https://github.com/apache/gravitino/pull/9271

Reply:
server/src/main/java/org/apache/gravitino/server/web/rest/TagOperations.java  Line 90.